groups/cvo/netapp.tf

```diff
@@ -33,6 +33,44 @@ module "cvo" {
 
   route_table_ids = [data.aws_route_table.private.route_table_id]
 
+  tags = merge(
+    local.default_tags,
+    map(
+      "ServiceTeam", "Storage"
+    )
+  )
+}
+
+module "cvo2" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/netapp_cloudmanager_cvo_aws?ref=tags/1.0.56"
+
+  vpc_id     = data.aws_vpc.vpc.id
+  subnet_ids = data.aws_subnet_ids.storage.ids
+
+  ## cvo setup
+  name                    = format("%s-%s-%s", var.cvo_name, var.account, "002")
+  instance_type           = var.cvo_instance_type
+  ebs_volume_size         = var.cvo_ebs_volume_size
+  ebs_volume_size_unit    = var.cvo_ebs_volume_size_unit
+  license_type            = var.cvo_license_type
+  is_ha                   = var.cvo_is_ha
+  use_latest_version      = false
+  ontap_version           = var.cvo_ontap_version
```

Comment:
You'll need to liaise with the storage team to ascertain what the updated version should be, then add a new variable for this. The second deployment must be running the updated software.